packages/react-native-reanimated/Common/cpp/worklets/Tools/defs.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+// JS_RUNTIME_HERMES is only set on Android so we have to check __has_include
+// on iOS.
```

Comment:
Let's mention in the comment that on Android, `JS_RUNTIME_HERMES` is set in CMakeLists.txt. On iOS, there's no simple way to detect if Hermes is enabled or not, so we check the existence of the header file.